tests/unit/plugins/modules/test_user.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020- IBM, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import unittest
+from unittest import mock
+import re
+import copy
+
+from ansible_collections.ibm.power_aix.plugins.modules import user
+
+from .common.utils import (
+    AnsibleFailJson, fail_json, rootdir
+)
+
+
+class Testcreate_user(unittest.TestCase):
+    def setUp(self):
+        self.module = mock.Mock()
+        self.module.fail_json = fail_json
+
+    def _setup_module_params(self):
+        self.module.run_command.return_value = (0, None, None)
+        params = dict()
+        params["name"] = "test123"
+        params["attributes"] = None
+        params["remove_homedir"] = True
+        params["change_passwd_on_login"] = True
+        params["password"] = "pass1234"
+        self.module.params = params
+
+    def test_success_create_user_without_password(self):
+        self._setup_module_params()
+        rc, stdout, stderr = 0, "sample stdout", "sample stderr"
```

Comment:
i would add 
```
rc, stdout, stderr = 0, "sample stdout", "sample stderr"
self.module.run_command.return_value = (rc, stdout, stderr)
```
this in the setUp method as the default `rc, stdout, stderr`
if you need this to change for a specific test case
then you can just overwrite it after the setUp method is ran at the
beginning of the test case